src/web-vitals.js

```diff
@@ -8,15 +8,16 @@ const METRIC_CONFIG = new Map([
   [
     'CLS',
     {
-      threshold: 0.1,
+      goodThreshold: 0.1,
+      needsImprovementThreshold: 0.25,
```

Comment:
Tiny nit here: what do you think of having a `threshold` property instead of these two.

```
{
  thresholds: {
    good: '',
    needsImprovement: '',
    poor: ''
  }
}
```

I think I'd like to avoid of having them all in the root object level. :)